packages/stackable/stackables/base.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+class BaseStackable {
+  constructor (options) {
+    this.stackableConfig = options.stackableConfig
+  }
```

Comment:
I would avoid using a class for this, as we can't guarantee uniqueness across versions,
but I can be convinced otherwise... so why is it a class and not just an "interface"?